frontend/src/components/CreateGoal/ObjectiveFiles.js

```diff
@@ -0,0 +1,102 @@
+import React, { useState, useEffect } from 'react';
+import PropTypes from 'prop-types';
+import {
+  FormGroup, Label, Radio,
+} from '@trussworks/react-uswds';
+import FileUploader from '../FileUploader';
+import QuestionTooltip from './QuestionTooltip';
+import './ObjectiveFiles.scss';
+
+export default function ObjectiveFiles({
+  objectiveId,
+  files,
+  onChangeFiles,
+  isOnApprovedReport,
+  status,
+}) {
+  const hasFiles = files && files.length > 0;
+  const [useFiles, setUseFiles] = useState();
+  const readOnly = isOnApprovedReport || status === 'Complete';
+
+  useEffect(() => {
+    if (hasFiles) {
+      setUseFiles(true);
+    }
+  }, [hasFiles]);
+  return (
+    <>
+      {
+      readOnly && hasFiles
+        ? (
+          <>
+            <p className="usa-prose margin-bottom-0 text-bold">Resources</p>
+            <p className="usa-prose margin-top-0">{files.map((f) => f.originalFileName).join(', ')}</p>
+          </>
+        )
+        : (
+          <FormGroup className="margin-top-1">
+            <Label>
```

Comment:
A label should be associated with something (a specific input). A group of radio buttons, however, is best marked up with a Fieldset/legend.

[Here is an example of the proper markup](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/radio)